examples/get-roles-mapping-report.py

```diff
@@ -0,0 +1,103 @@
+# Requires Python 3.6+
+import sys
+import csv
+
+# Import the PrivX python library.
+import privx_api
+
+
+# Import the configs.
+import config
+
+# Initialize the API.
+api = privx_api.PrivXAPI(config.HOSTNAME, config.HOSTPORT, config.CA_CERT,
+                         config.OAUTH_CLIENT_ID, config.OAUTH_CLIENT_SECRET)
+
+# Authenticate.
+# NOTE: fill in your credentials from secure storage, this is just an example
+api.authenticate("API client ID", "API client secret")
+
+
+def get_roles():
+    resp = api.get_roles()
+    if resp.ok():
+        data_load = resp.data()
+        roles_data = {}
+        for role_data in data_load['items']:
+            name = role_data['name']
+            roles_data[name] = role_data['id']
+        return roles_data
+    else:
+        error = "Get Roles operation failed:"
+        process_error(error)
+
+
+def get_role_mapping_data(role_id, role_name):
+    global header_printed
```

Comment:
I think this is unnecessary now? It doesn't seem to be used in this function.

Reply:
removed it now.